### PR TITLE
feat: handle focusing of interactive elements inside of a Flyout

### PR DESCRIPTION
**Pull request by noahwaldner**
handle focusing of interactive elements inside of a Flyout

- Don't show focus visible when opened with mouse

### CODE REVIEW

File: packages/components/src/components/Flyout/__tests__/Flyout.ct.tsx [hunk 1]
```diff
@@ -364,3 +368,81 @@ test('should trigger callback on open and close', async ({ mount, page }) => {
...
+        'box-shadow',
+        'rgb(255, 255, 255) 0px 0px 0px 2px, rgb(65, 150, 251) 0px 0px 0px 6px',
+    );
+    await page.keyboard.press('Tab');
+    await expect(textInput2).toHaveCSS(
+        'box-shadow',
+        'rgb(255, 255, 255) 0px 0px 0px 2px, rgb(65, 150, 251) 0px 0px 0px 6px',
+    );
+    await textInput3.click();
+    await expect(textInput3).not.toHaveCSS(
```
Comment (jose-costa-frontify) [inline packages/components/src/components/Flyout/__tests__/Flyout.ct.tsx:405]: use these thingies https://github.com/Frontify/fondue/blob/main/packages/components/src/helpers/constants.ts

File: packages/components/src/components/Flyout/Flyout.tsx [hunk 2]
```diff
@@ -113,6 +122,16 @@ export const FlyoutContent = (
                 className={flyoutContentStyles({ ...props })}
                 data-flyout-spacing={padding}
                 data-test-id={dataTestId}
+                onFocus={(e) => {
```
Comment (jose-costa-frontify) [inline packages/components/src/components/Flyout/Flyout.tsx:125]: I think we don't use initials for variable names

**Review by jose-costa-frontify — CHANGES_REQUESTED** [review]
really good stuff 🚀

File: packages/components/src/components/Flyout/Flyout.tsx (hunk 2 above)
Comment (noahwaldner) [reply packages/components/src/components/Flyout/Flyout.tsx]: fixed

File: packages/components/src/components/Flyout/__tests__/Flyout.ct.tsx (hunk 1 above)
Comment (noahwaldner) [reply packages/components/src/components/Flyout/__tests__/Flyout.ct.tsx]: adjusted

**Review by jose-costa-frontify — APPROVED** [review]
LGTM

**Pull request merged**